extensions/wikidata/src/org/openrefine/wikidata/qa/WikidataConstraintFetcher.java

```diff
@@ -28,9 +28,7 @@
 import org.wikidata.wdtk.datamodel.implementation.QuantityValueImpl;
 import org.wikidata.wdtk.datamodel.interfaces.*;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
Did your IDE do this without you asking? Can you configure a higher limit for wild card imports and/or review the diff to make sure unnecessary changes aren't included?

Reply:
Yes, it got updated on its own by the IDE. Sorry, I'll take care of it next time onwards for sure. For now, I'll create a PR resolving all the flaws. Thanks for the review. :)